src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -696,4 +671,70 @@ describe('workspace service api integration test when savedObjects.permission.en
       ).toEqual({ write: { users: ['foo'] } });
     });
   });
+
+  describe('Saved Object Management APIs', () => {
```

Comment:
Shall we move these integration tests under `src/plugins/saved_objects_management` folder? It was testing saved object management APIs instead of workspace related API. I'm prefer to creating integration tests in that file.

Reply:
This is a bit tricky I guess, because here we only test the saved object API under path `w/${workspaceId}/saved_object...`, which means we only test the API under workspace context. This is to ensure the behaviour of the API is expected when we already entered a workspace and trying to find saved objects. Therefore I think it is more relevant to the workspace scope instead of saved object management context.